tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -1085,6 +1085,9 @@ async def tracked_prime_for_state_transfer(
             kv = [(key, val)]
             await self.write_and_track_known_kv(kv, client1)
 
+        await self.bft_network.wait_for_replicas_to_collect_stable_checkpoint(self.bft_network.get_live_replicas(),
```

Comment:
Maybe use network_wait_for_checkpoint?

Reply:
I checked the code and **network_wait_for_checkpoint** waits for the **last_stored_checkpoint** metric in Replicas to get to the needed value. This metric is updated when we generate a Checkpoint Message. In order to shift the Working Window we need the Replicas to collect a Stable Checkpoint.